tests/unit/test_install.py

```diff
@@ -89,24 +85,59 @@ def test_save_config(ws, mock_installation):
             "version": 1,
             "source": "snowflake",
             "skip_validation": True,
-            "catalog_name": "remorph_catalog",
-            "schema_name": "remorph_schema",
+            "catalog_name": "transpiler_test",
+            "schema_name": "convertor_test",
```

Comment:
Please refer to the comment on moving the default catalog and schema names to constants.

Reply:
Created a technical debt to move this to constants